database/migrations/2015_09_22_003413_migrate_mac_address.php

```diff
@@ -55,6 +55,6 @@ public function down()
         Schema::table('models', function (Blueprint $table) {
             $table->renameColumn('deprecated_mac_address', 'show_mac_address');
         });
-        DB::statement('ALTER TABLE assets CHANGE _snipeit_mac_address mac_address varchar(255)');
+        DB::statement('ALTER TABLE assets CHANGE _snipeit_mac_address_1 mac_address varchar(255)');
```

Comment:
This isn't ideal and will only work on fresh installs. I considered dropping all `_snipeit_*` columns from this table via the method used in [`CustomFieldSeeder`](https://github.com/snipe/snipe-it/blob/4f8951f6f148071853df187ca7e7b8fa991e6970/database/seeders/CustomFieldSeeder.php#L16-L24) but I'm hesitant since it seems out of scope for this migration _but_ this is holding up being able to run `migrate:rollback` completely.

This fix works now but I think we will want to revisit it in the future.

Reply:
@marcusmoore - this migration works as intended - we rename that column further down in the migrations, IIRC

Reply:
For additional context, the `_snipeit_*` columns are custom fields. In the before times, we had mac address as a regular field, but once custom fields were introduced, we switched this field to be a custom field that's "pre-built". We renamed it since *some* folks were using it and some folks weren't, and (obviously) had to retain that data.

Reply:
I took a quick stab at trying to find where we rename `_snipeit_mac_address` to `_snipeit_mac_address_1` but I wasn't able to figure out where it is :/

I just took another stab at finding it, actually - and I still can't figure out where it is! But we could make a good argument that, wherever that might be, that should be the place where we pull the `_1` so that this migration (as-is) would continue to work. The 'migration' might just be something that changed in the `CustomField` class itself? But I can't see how that might've renamed the column (off the top of my head)

Another fair argument to make (which might be the same as this one? Or slightly different) would be that when you `down` from having-custom-fields to not-having-custom-fields could be that you should drop *ALL* columns with `_snipeit_*` - which I totally understand your reticence in doing, but could make sense for us here.

Reply:
It took a bit but I tracked it down to `2017_01_25_063357_fix_utf8_custom_field_column_names.php`. I haven't investigated that migration after figuring out it was the one doing the change yet though.